File: java/filtering-content/GetItemsByTaxonomyTerm.java
// DocSection: filtering_get_items_by_taxonomy_term
// Note: Filters work with codenames of the tags.
// Gets items tagged with one specific tag
CompletionStage<ContentItemsListingResponse> items = client.getItems(
  DeliveryParameterBuilder.params()
    .filterContains("elements.tags", "kentico")
    .build()
);

// Gets items tagged with a list of specific tags
CompletionStage<ContentItemsListingResponse> items = client.getItems(
  DeliveryParameterBuilder.params()
    .filterAll("elements.tags", "kontent", "headless")
    .build()
);

// Gets items tagged with at least one of multiple tags
CompletionStage<ContentItemsListingResponse> items = client.getItems(
  DeliveryParameterBuilder.params()
    .filterAny("elements.tags", "football", "soccer")
    .build()
);
// EndDocSection

File: js/filtering-content/GetItemsByTaxonomyTerm.js
// DocSection: filtering_get_items_by_taxonomy_term
deliveryClient.items()
  // Get articles tagged with specific tag
  .containsFilter("elements.tags", ["kentico"])

deliveryClient.items()
  /// Gets articles tagged with any of the below tags
  .anyFilter("elements.tags", ["football"], ["soccer"])

deliveryClient.items()
  // Gets articles tagged with all of the below tags
  .allFilter("elements.tags", ["mvc"], ["kontent"], ["headless"])
// EndDocSection
